### ISSUE TITLE
Save column means for imputation on test set

### ISSUE THREAD

**Issue by sftwre** [sftwre/glm_api#9]
When np.NaN are encountered in API calls with a single row, mean imputation will drop the column. We need to maintain the columns means on the training data to properly fill empty features.
